article/app/controllers/LiveBlogController.scala

```diff
@@ -55,17 +55,24 @@ class LiveBlogController(
     }
   }
 
-  def renderArticle(path: String, page: Option[String] = None, filterKeyEvents: Option[Boolean]): Action[AnyContent] = {
+  def renderArticle(
+      path: String,
+      page: Option[String] = None,
+      filterKeyEvents: Option[Boolean],
+      automaticFilter: Option[String],
+  ): Action[AnyContent] = {
     Action.async { implicit request =>
       val filter = shouldFilter(filterKeyEvents)
+      val topMentionResult = getTopMentionsForFilters(path, automaticFilter)
+
       page.map(ParseBlockId.fromPageParam) match {
         case Some(ParsedBlockId(id)) =>
-          renderWithRange(path, PageWithBlock(id), filter) // we know the id of a block
+          renderWithRange(path, PageWithBlock(id), filter, topMentionResult) // we know the id of a block
```

Comment:
Not too familiar with top mentions, but would you expect to be able to have and use both the blockId and the automaticFilter parameters in the same request?

Reply:
when we filter the blocks by the provided filters, we could end up with several pages. So we would still need to be able to handle the pages when a page id is provided in the url